starport/cmd/scaffold_map.go

```diff
@@ -0,0 +1,35 @@
+package starportcmd
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/tendermint/starport/starport/services/scaffolder"
+)
+
+const (
+	flagModule    = "module"
+	flagNoMessage = "no-message"
+)
+
+// NewScaffoldMap returns a new command to scaffold a map.
+func NewScaffoldMap() *cobra.Command {
+	c := &cobra.Command{
+		Use:   "map NAME [field]...",
+		Short: "Scaffold a map",
+		Args:  cobra.MinimumNArgs(1),
+		RunE:  scaffoldMapHandler,
+	}
+
+	c.Flags().StringVarP(&appPath, "path", "p", "", "path of the app")
+	c.Flags().AddFlagSet(flagSetScaffoldType())
+
+	return c
+}
+
+func scaffoldMapHandler(cmd *cobra.Command, args []string) error {
+	opts := scaffolder.AddTypeOption{
+		Indexed:   true,
```

Comment:
This should be refactored. Also this is not part of the PR since this one is only about CLI interface change. I can handle this in `single` scaffolding